packages/npm-dependencies-validation/src/utils/npmUtil.ts

```diff
@@ -0,0 +1,26 @@
+import { spawn } from "child_process";
+
+export function getNPM(): string {
+  return /^win/.test(process.platform) ? "npm.cmd" : "npm";
+}
+
+export function spawnCommand(
```

Comment:
`SpawnCommand` name is not the most descriptive.

- It is **implemntation** details that we use `spawn`
 - maybe we will swith to `exec`?
 - maybe we will switch to programatic api?

Maybe `SpawnCommand` --> `invokeNPMCommand` ?
- can replace `invoke` with `run` / `perform` / ???

Reply:
although, maybe because we expose `  commandArgs: string[],` param
we can't really **abstract* away that a child process is run here.

Reply:
Done